typings/styled-components.d.ts

```diff
@@ -199,6 +199,8 @@ export interface ThemedStyledComponentsModule<T> {
 declare const styled: StyledInterface;
 
 export const css: ThemedCssFunction<any>;
+export function withTheme<C extends Component<any>>(component: C): C;
```

Comment:
1. what's the point for this? you need to specify the theme prop anyway, like this:

```ts
class MyComponent extends React.Component<ThemeProps<{}>, {}> {
  render() {
    const { theme } = this.props;

    console.log("Current theme: ", theme);

    return <h1>Hello</h1>;
  }
}

export default withTheme(MyComponent);
```
2. yeah, that would be helpful, will change :)
3. same as 1
4. will change